Tasks/AppCenterDistribute/task.json

```diff
@@ -11,8 +11,8 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 0,
-        "Minor": 131,
+        "Major": 1,
+        "Minor": 134,
         "Patch": 0
     },
```

Comment:
Add "relaseNotes" to indicate to the user whats new or different in 1.* from 0.*. E.g. https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/Maven/task.json#L22

Reply:
Sure, thanks!

Reply:
Added.